omc3/utils/time_tools.py

```diff
@@ -101,17 +107,20 @@ def __new__(cls, *args, **kwargs):
             dt = args[0]
         else:
             dt = datetime.__new__(cls, *args, **kwargs)
-        dt = utc_to_local(dt, pytz.utc)  # does not convert, but checks tz
+
+        if dt.tzinfo == None:
```

Comment:
would be better to use `is None` here, as `== None` can sometimes return a non-boolean